yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -484,7 +484,7 @@ func convertAssessmentIssueToYugabyteDAssessmentIssue(ar AssessmentReport) []Ass
 func runAssessment() error {
 	log.Infof("running assessment for migration from '%s' to YugabyteDB", source.DBType)
 
-	err := migassessment.SizingAssessment(targetDbVersion)
+	err := migassessment.SizingAssessment(targetDbVersion, &source.DBType)
```

Comment:
You can also pass the db type by value only, right? Is there an issue with that?

Reply:
done, updated code to pass by value in recent commit